content/upgrade_server.md

```diff
@@ -153,90 +145,100 @@ For the latest information on high availability and how to set up a highly-avail
 
     ```bash
     rpm -Uvh --nopostun /path/to/chef-server-core-<version>.rpm
     ```
 
-1.  Stop the back end server:
+1. Stop the back end server:
 
     ```bash
     chef-server-ctl stop
     ```
 
-1.  Upgrade the backend server with the following command. You will need to type `Yes` to accept Chef License for version 13.
+1. Upgrade the server and accept the Chef Software license by entering `Yes` at the prompt:
 
     ```bash
     chef-server-ctl upgrade
     ```
 
-    To accept the license and upgrade to version 14 in one command:
+    To accept the license and upgrade in one command:
 
     ```bash
     CHEF_LICENSE='accept' chef-server-ctl upgrade
     ```
 
-1.  Copy the entire `/etc/opscode` directory from the back end server to all front end servers:
+1. Copy the entire `/etc/opscode` directory from the back end server to all front end servers:
 
-    ```none
+    ```bash
     scp -r /etc/opscode <each server's IP>:/etc
     ```
 
-1.  Upgrade each of the front end servers:
+1. Upgrade each of the front end servers:
 
     ```bash
     chef-server-ctl upgrade
     ```
 
-1.  Run the following command on both the front end, and back end servers:
+1. Run the following command on both the front end, and back end servers:
 
     ```bash
     chef-server-ctl start
     ```
 
-1.  [Upgrade](#upgrading-add-ons) the Chef Infra Server add-ons on each server.
+1. [Upgrade](#upgrading-add-ons) any Chef Infra Server add-ons.
 
-1.  After the upgrade process is complete, the state of the system after the upgrade has been tested and verified, and everything looks satisfactory, remove old data, services, and configuration by running the following command on each server:
+1. After the upgrade process is complete, test and verify that the server works properly.
 
-    ```bash
-    chef-server-ctl cleanup
-    ```
+1. Clean up the server by removing the old data:
+
+   ```bash
+   chef-server-ctl cleanup
+   ```
 
 ## Upgrading Add-ons
 
-This section details the process for upgrading additional features after the Chef Infra Server has been upgraded.
+Chef Infra Server 13 and 14 supports Chef Manage and push jobs. Both of these add-ons are deprecated and will reach [EOL](https://docs.chef.io/versions/#deprecated-products-and-versions) on December 31, 2021. After upgrading Chef Infra Server, reinstall the add-on and then reconfigure Chef Infra Server and the add-on.
+
+Chef Manage
+: Chef Manage is deprecated and will reach [EOL](https://docs.chef.io/versions/#deprecated-products-and-versions) on December 31, 2021. Chef Manage is a management console for  data bags, attributes, run-lists, roles, environments, and cookbooks from a web user interface
+
+Push jobs
+: Push jobs deprecated and will reach [EOL](https://docs.chef.io/versions/#deprecated-products-and-versions) on December 31, 2021. Chef Push Jobs is an extension of the Chef Infra Server that allows for running jobs against nodes independently of a Chef Infra Client run.
```

Comment:
```suggestion
: Chef Push Jobs is deprecated and will reach [EOL](https://docs.chef.io/versions/#deprecated-products-and-versions) on December 31, 2021. Chef Push Jobs is an extension of the Chef Infra Server will run jobs against nodes independently of a Chef Infra Client run.
```

Reply:
It is just "push jobs", but I am unsure about the capitalization.

Reply:
Went with "Push Jobs"